code/modules/tgs_daedalus/tgs_http_rustg.dm

```diff
@@ -0,0 +1,22 @@
+/datum/tgs_http_handler/rustg
+
+/datum/tgs_http_handler/rustg/PerformGet(url)
+	var/datum/http_request/request = new()
+	request.prepare(RUSTG_HTTP_METHOD_GET, url)
+	request.begin_async()
+
+	TGS_DEBUG_LOG("http_rustg: Awaiting response.")
+	UNTIL(request.is_complete())
+	TGS_DEBUG_LOG("http_rustg: Request complete!")
```

Comment:
Don't use `TGS_DEBUG_LOG` it's a define that evaluates to nothing unless you set it. And then it's _really_ noisy

Reply:
Really it's only used by the TGS integration tests (NOT the ones used in SS13 repositories)